terragrunt/env/staging/security_hub/terragrunt.hcl

```diff
@@ -0,0 +1,7 @@
+terraform {
```

Comment:
Could you add the `env/production/security_hub/terragrunt.hcl` along with the `.terraform.lock.hcl` files for both.